include/deal.II/base/vectorization.h

```diff
@@ -526,6 +527,13 @@ class VectorizedArray
     data = base_ptr[offsets[0]];
   }
 
+  DEAL_II_ALWAYS_INLINE
```

Comment:
missing documentation?

Reply:
You are right. I have forgotten it. Now its there.